core/base/mergeTreeClustering/MergeTreeDistance.h

```diff
@@ -0,0 +1,1504 @@
+/// \ingroup base
+/// \class MergeTreeDistance
+/// \author Mathieu Pont (mathieu.pont@lip6.fr)
+/// \date 2021.
+///
+/// This module defines the %MergeTreeDistance class that computes distance
+/// between two merge trees.
+///
+/// \b Related \b publication \n
+/// "Wasserstein Distances, Geodesics and Barycenters of Merge Trees" \n
+/// Mathieu Pont, Jules Vidal, Julie Delon, Julien Tierny.\n
+/// Proc. of IEEE VIS 2021.\n
+/// IEEE Transactions on Visualization and Computer Graphics, 2021
+
+#ifndef _MERGETREEDISTANCE_H
+#define _MERGETREEDISTANCE_H
+
+#pragma once
+
+#include <stack>
+#include <thread>
+
+// ttk common includes
+#include <Debug.h>
+#include <Triangulation.h>
```

Comment:
are you using the triangulation?
(if not, this may be removed)

Reply:
Thanks! Done in the next commit.